main.tf

```diff
@@ -60,6 +60,33 @@ locals {
 
   subnet_name_pod_cidr     = "gitlab-cluster-pod-cidr"
   subnet_name_service_cidr = "gitlab-cluster-service-cidr"
+
+  default_pool = [{
+    name                       = "gitlab"
+    description                = "Gitlab Cluster"
+    machine_type               = var.gke_machine_type
+    node_count                 = 1
+    min_count                  = var.gke_min_node_count
+    max_count                  = var.gke_max_node_count
+    disk_size_gb               = 100
+    disk_type                  = "pd-balanced"
+    image_type                 = "COS_CONTAINERD"
```

Comment:
we should let this values be changeable, please convert them into variables with the same default

Reply:
Done